tests/features_test_files/formatting/simple_input.vv

```diff
@@ -0,0 +1,10 @@
+module main
```

Comment:
shouldn't we follow the scheme of V with `simple.vv` and `simple.out`?

Reply:
I used vfmt test as a basis when I wrote the test. I'll rename it later😀

Reply:
Fixed :)